src/nupic/engine/Input.hpp

```diff
@@ -260,6 +260,26 @@ class Input {
    * cached splitter map -- only created if requested
    * mutable because getSplitterMap() is const and logically
    * getting the splitter map doesn't change the Input
+   *
+   * What is a SplitterMap?
```

Comment:
Thanks! These detailed explanations are very valuable for future-proof and new developers! :+1: